js/languages/julia/Makefile

```diff
@@ -1,3 +1,5 @@
 SEMGREP_LANG := julia
 
+EMCC_TEST_OPTIMIZATION := -O1
```

Comment:
do we still want this now that we have the LLVM fix?

Reply:
So the "fix" makes it so that things break at higher optimization levels... and lower... depending on the language... That's why I made it opt in for only specific lanugages, and require an opt level.